src/main/webapp/dev/AdminActivityLogTest.js

```diff
@@ -0,0 +1,129 @@
+    $.fx.off = true;
+
+    // show reference
+    toggleReference();
+    assert.ok($filterReferenceDiv.is(':visible'), 'filterReference div should be visible');
+    assert.equal($detailButton.attr('class'), 'glyphicon glyphicon-chevron-up');
+    assert.equal($referenceLink.text(), 'Hide Reference', 'Reference links text should change appropriately');
+
+    // hide reference
+    toggleReference();
+    assert.notOk($filterReferenceDiv.is(':visible'), 'filterReference div should be hidden');
+    assert.equal($detailButton.attr('class'), 'glyphicon glyphicon-chevron-down');
+    assert.equal($referenceLink.text(), 'Show Reference', 'Reference links text should change appropriately');
+
+    $.fx.off = false;
+});
+
+QUnit.test('updatePageWithNewLogsFromAjax(response, selector)', function(assert) {
+    var requiredElements = '<table id="logsTable"> <tbody> </tbody> </table>';
+    createRequiredElements(requiredElements);
+    var selector = '#logsTable tbody';
+    var logContainer = $(selector);
+
+    var responseWithNoLogs = { logs: [] };
+    updatePageWithNewLogsFromAjax(responseWithNoLogs, selector);
+    assert.equal(logContainer.children().length, 0, 'no entries should be added if ajax response contains no entries');
+
+    var responseWithLogs = { logs: [
+                                    { logInfoAsHtml: '<tr><td>entry 1</td></tr>' },
+                                    { logInfoAsHtml: '<tr><td>entry 2</td></tr>' }
+    ] };
+    updatePageWithNewLogsFromAjax(responseWithLogs, selector);
+    assert.equal(logContainer.children().length, 2, 'Log entries should be added');
+    assert.equal(logContainer.children().eq(0).html(), '<td>entry 1</td>');
+    assert.equal(logContainer.children().eq(1).html(), '<td>entry 2</td>');
+});
+
+var $ajaxImplementation = $.ajax;
+
+/**
+ * Replaces $.ajax with a stub that executes the success callback synchronously with a simulated response.
+ *
+ * @param {Object} simulatedResponse to use with success callback
+ */
+function replaceAjaxWithStub(simulatedResponse) {
+    $.ajax = function(opts) {
+        var successCallback = opts.success;
+        successCallback(simulatedResponse);
+    };
+}
+
+function restoreAjaxImplementation() {
+    $.ajax = $ajaxImplementation;
+}
+
+QUnit.test('getOlderLogEntriesByAjax correctly modifies page when receiving new log entries', function(assert) {
+    var requiredElements = '<form id="ajaxLoaderDataForm"> <input type="hidden" name="searchTimeOffset" value=""> </form> '
+        + '<table id="logsTable"> <tbody> </tbody> </table>'
+        + '<button id="button_older">Search More</button>';
+    createRequiredElements(requiredElements);
+    var selector = '#logsTable tbody';
+    var logContainer = $(selector);
+
+    var responseWithNoLogs = {
+        isError: false,
+        logs: []
+    };
+    replaceAjaxWithStub(responseWithNoLogs);
+    getOlderLogEntriesByAjax(1488712272839);
```

Comment:
Could you give this magic number a name? Applies for the other epoch times as well.